redshift_sqlalchemy/dialect.py

```diff
@@ -320,67 +323,151 @@ def visit_unload_from_select(element, compiler, **kw):
             'parallel': element.options.get('parallel', 'ON')}
 
 
+creds_rx = re.compile(r"""
+    ^aws_access_key_id=[A-Z0-9]{20};
+    aws_secret_access_key=[A-Za-z0-9/+=]{40}
+    (?:;token=[A-Za-z0-9/+=]+)?$  # TODO: regex for token
```

Comment:
Can you change the comment from the TODO to one that discusses your findings instead?